assets/css/_pages.scss

```diff
@@ -1,3 +1,33 @@
 span.highlight {
     background-color: yellow;
 }
+
+.historical-records {
+  table {
+    border-collapse: collapse;
```

Comment:
Properties should be ordered width, margin, table-layout, border-collapse

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

Reply:
gtg